g4py.sh

```diff
@@ -0,0 +1,51 @@
+package: G4PY
+version: "%(tag_basename)s%(defaults_upper)s"
+source: https://github.com/alisw/geant4
+tag: v4.10.01.p03
+requires:
+  - GEANT4
+  - Python-modules
+  - ROOT
+  - boost
+  - XercesC
+  - opengl
+build_requires:
+  - CMake
+  - "Xcode:(osx.*)"
+env:
+  G4PYINSTALL: "$G4PY_ROOT"
+---
+#!/bin/bash -e
+cp -a $SOURCEDIR/environments/g4py/* $INSTALLROOT
```

Comment:
We are frequently in contact with GEANT4 developers - quite evidently, the behaviour is wrong, and if you have a modified GEANT4 source that works, the best option is to open a pull request. They are very, very responsive (especially if you tag us!)

Anyway we'll see the results from your test... thanks!